app/controllers/menus_controller.rb

```diff
@@ -2,6 +2,11 @@ class MenusController < ApplicationController
 
   def show
     @restaurant = Restaurant.find(params[:restaurant_id])
-    @cart_nr = ShoppingCartItem.count
+    if session[:cart_id]
+      @cart = ShoppingCart.find(session[:cart_id])
+      @cart_nr = @cart.shopping_cart_items.count
```

Comment:
I didn't mess with wherever it is implemented because it would be out of scope. The number as it was just returned the total items in all shopping carts, ever. So we had to fix it so it actually returned the intended amount. This number is used somewhere to display `# items in shopping cart`